site/content/docs/5.2/utilities/position.md

```diff
@@ -94,7 +94,7 @@ Here are some real life examples of these classes:
 </button>
 
 <button type="button" class="btn btn-dark position-relative">
-  Marker <svg width="1em" height="1em" viewBox="0 0 16 16" class="position-absolute top-100 start-50 translate-middle mt-1" fill="#212529" xmlns="http://www.w3.org/2000/svg"><path d="M7.247 11.14L2.451 5.658C1.885 5.013 2.345 4 3.204 4h9.592a1 1 0 0 1 .753 1.659l-4.796 5.48a1 1 0 0 1-1.506 0z"/></svg>
+  Marker <svg width="1em" height="1em" viewBox="0 0 16 16" class="position-absolute top-100 start-50 translate-middle mt-1" fill="inherit" xmlns="http://www.w3.org/2000/svg"><path d="M7.247 11.14L2.451 5.658C1.885 5.013 2.345 4 3.204 4h9.592a1 1 0 0 1 .753 1.659l-4.796 5.48a1 1 0 0 1-1.506 0z"/></svg>
```

Comment:
This change doesn't work in dark mode and seemed to be better in dark mode:

![Screenshot 2022-11-18 at 18 29 33](https://user-images.githubusercontent.com/17381666/202765988-1fd7a8f8-adf5-47ff-b922-d3897a42d238.png)

Reply:
Have a fix here that just cleans up the broke stuff, but doesn't invert it for dark mode. Will need to experiment more with that.